schema.graphql

```diff
@@ -419,6 +419,43 @@ type TransferManager @entity {
   updatedBlock: Block!
 }
 
+type StatType @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset!
+  opType: String!
+  claimType: String
+  claimIssuer: Identity
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+type TransferCompliance @entity {
+  id: ID! # ticker/op/type/claimIssuer
+  asset: Asset! @index(unique: false)
+  type: String!
+  statType: StatType!
+  # value is present only in case of MaxInvestorCount and MaxInvestorOwnership
+  value: BigInt
+  # following values are present in case of ClaimCount and ClaimOwnership
+  claimType: String
+  claimValue: String
+  claimIssuer: Identity
+  min: BigInt
+  max: BigInt
+  createdBlock: Block!
+  updatedBlock: Block!
+}
+
+type TransferComplianceExemption @entity {
+  id: ID! #asset/opType/claimType/exemptedEntityId
```

Comment:
Could this have a reference to the TransferCompliance entity?

To me it seems like it would be hard to figure out who is exempted from a condition now.

Reply:
I don't think so we can have any relationship there. The exemption are tagged with an ExemptKey which is combination of `assetId + opType + claimType` and transfer compliance are tagged with `StatType` which may or may not depend on issuerId, which we wouldn't have in case of exempted entities. Do you have a way around this?

Reply:
Hmm, I think you are right. In the SDK we make it seem like the exemptions are part of the restriction itself, but on chain there isn't a relationship like that.

I guess this will have to do.